x/gov/spec/03_messages.md

```diff
@@ -16,8 +16,6 @@ must be registered in the app's `MsgServiceRouter`. Each of these messages must
 have one signer, namely the gov module account. And finally, the metadata length
 must not be larger than the `maxMetadataLen` config passed into the gov keeper.
 
-NOTE DIFF FROM UPSTREAM: All proposals must have at least one `sdk.Msg` in the `messages` field.
-
```

Comment:
Note that this will now be enforced on celestia-app here: https://github.com/celestiaorg/celestia-app/blob/cb8247ead807a52bb8915d450982544755efc1e6/app/ante/gov.go#L17